features/call/test/src/main/kotlin/io/element/android/features/call/test/FakeCurrentCallObserver.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2024 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ * Please see LICENSE in the repository root for full details.
+ */
+
+package io.element.android.features.call.test
+
+import io.element.android.features.call.api.CurrentCall
+import io.element.android.features.call.api.CurrentCallObserver
+import kotlinx.coroutines.flow.MutableStateFlow
+
+class FakeCurrentCallObserver(
+    initialValue: CurrentCall = CurrentCall.None,
+) : CurrentCallObserver {
+    override val currentCall = MutableStateFlow(initialValue)
```

Comment:
Usually when we have `StateFlow` we provide them in the constructor of the fake

Reply:
db4b4d3fa4e3434e13a2c7af4ee7abb578304c5f